projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -81,6 +81,11 @@ so we have to target all affected elements in touch devices.
 	border-radius: 4px;
 }
 
+.jp-carousel-overlay .swiper-button-prev svg:hover,
+.jp-carousel-overlay .swiper-button-next svg:hover {
+	background: var(--jp-carousel-primary-subtle-color);
+}
```

Comment:
There is a weid mismatch of hover states between the svg and the wrapping div:

https://user-images.githubusercontent.com/8486249/208675698-90cc0395-769c-4cc5-92f1-40584103256e.mp4

The svg seems to be correct though. The click area of the buttons extends around the icon for some reason:

https://user-images.githubusercontent.com/8486249/208677170-108aa3c0-db60-4598-bd80-a05bd6908c9e.mp4